src/backoff.erl

```diff
@@ -8,13 +8,14 @@
 -record(backoff, {start :: pos_integer(),
                   max :: pos_integer() | infinity,
                   current :: pos_integer(),
-                  type=normal :: normal | jitter,
+                  type=normal :: type(),
```

Comment:
I have force pushed a change so that `type()` remains but in the record definition the atoms are explicitly listed. However `type()` is still used in the specs of `type/2` and is still exported.